apollo-sample/src/main/java/com/example/apollographql/sample/MainActivity.java

```diff
@@ -18,25 +21,31 @@
 public class MainActivity extends AppCompatActivity {
   private static final String TAG = "MainActivity";
 
+  private SampleApplication application;
+
   @Override protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
     final TextView txtResponse = (TextView) findViewById(R.id.txt_response);
-    SampleApplication application = (SampleApplication) getApplication();
-    application.githuntApiService()
-        .githuntFeed(new FeedQuery(FeedQuery.Variables.builder()
-                .limit(10)
-                .type(FeedType.HOT)
-                .build()))
+    application = (SampleApplication) getApplication();
+
+    Observable<Response<FeedQuery.Data>> query = Observable.fromCallable(new Callable<Response<FeedQuery.Data>>() {
```

Comment:
Actually theres no need to even do rxjava anymore. there is an apolloclient method .enque that takes a callback and schedules to run in background... use that instead and drop rxjava from sample please